### ISSUE TITLE
Conceptual connections to Mozilla's "optimized zip"

### ISSUE THREAD

**Issue by vadimkantorov** [sozip/sozip-spec#13]
I wonder if you have stumbled on https://taras.glek.net/post/optimized-zip-format/ . Are the stated goals of SOZip and "optimized zip" similar or different?

"optimized zip" stresses suitability to OS's read-ahead functionality and sequential scan and SOZip seems to stress perf of random access, so it seems different goals to me, but maybe I'm mistaken.

Thanks!

**Comment by rouault** [comment]
No, Mozilla's optimized zip is a trick to move the "file directory" at the beginning of the zip file, and place frequently accessed files toward the beginning.
SOZip goal is to help you to quickly access any part of a typically gigabyte-large file compressed in a ZIP.

Both optimizations are actually "orthogonal", i.e. you could combine them
